content/github/collaborating-with-issues-and-pull-requests/about-status-checks.md

```diff
@@ -53,16 +53,16 @@ When a repository is set to automatically request checks for pushes, you can cho
 
 To skip or request checks for your commit, add one of the following trailer lines to the end of your commit message:
 
-- To _skip checks_ for a commit, type your commit message and a short, meaningful description of your changes. After your commit description, instead of a closing quotation, add two empty lines followed by `skip-checks: true`:
+- To _skip checks_ for a commit, type your commit message and a short, meaningful description of your changes. After your commit description, instead of a closing quotation, add two empty lines followed by `skip-checks: true`":
```

Comment:
My bad for not making this clearer in the issue description 😄 

```suggestion
- To _skip checks_ for a commit, type your commit message and a short, meaningful description of your changes. After your commit description, instead of a closing quotation, add two empty lines followed by `skip-checks: true"`:
```

Reply:
But that makes me realize that this would be clearer:

```suggestion
- To _skip checks_ for a commit, type your commit message and a short, meaningful description of your changes. After your commit description, before the closing quotation, add two empty lines followed by `skip-checks: true`:
```

Let me know if you think otherwise!